app/scripts/modules/core/cluster/filter/clusterFilter.service.js

```diff
@@ -276,7 +277,7 @@ module.exports = angular
 
         groups.push( {
           heading: accountKey,
-          subgroups: _.sortBy(clusterGroups, 'heading')
+          subgroups: _.sortBy(clusterGroups, ['heading', 'category']),
```

Comment:
I thought it was `cluster.category` based on the original PR (but I guess they're going to be the same value, eh)